src/Orleans.Core/Configuration/GlobalConfiguration.cs

```diff
@@ -1036,35 +1036,6 @@ internal override void Load(XmlElement root)
             }
         }
 
-        /// <summary>
-        /// Registers a given type of <typeparamref name="T"/> where <typeparamref name="T"/> is bootstrap provider
-        /// </summary>
-        /// <typeparam name="T">Non-abstract type which implements <see cref="IBootstrapProvider"/> interface</typeparam>
-        /// <param name="providerName">Name of the bootstrap provider</param>
-        /// <param name="properties">Properties that will be passed to bootstrap provider upon initialization</param>
-        public void RegisterBootstrapProvider<T>(string providerName, IDictionary<string, string> properties = null) where T : IBootstrapProvider
```

Comment:
Legacy configuration class should stay unchanged, right? What's the reason of removing this two methods from it ?